.babelrc

```diff
@@ -0,0 +1,4 @@
+{
+  "presets": ["es2015", "react"],
+  "plugins": [["react-templates", {"ext": ".rt.html", "modules": "commonjs"}]]
+}
```

Comment:
Since this repo won't actually contain the app, we won't need react or react-templates plugins.